App/Lettuce/Sources/Lettuce/UI/MessagesDetailChatView/MessageDetailNavigationView.swift

```diff
@@ -118,31 +113,33 @@ struct MessageDetailNavigationView: View {
     private struct MessageBubble: View {
 
         @Environment(\.colorScheme) private var colorScheme
+        private let messageText: String
         private let isInterlocutor: Bool
         private let geometry: GeometryProxy
         private let alignment: Alignment
         private let horizontalAlignment: HorizontalAlignment
         private let cornerRadius: CGFloat = 18
 
-        init(geometry: GeometryProxy, isInterlocutor: Bool) {
+        init(geometry: GeometryProxy, isInterlocutor: Bool, messageText: String) {
             self.geometry = geometry
             self.isInterlocutor = isInterlocutor
             self.alignment = isInterlocutor ? .leading : .trailing
             self.horizontalAlignment = isInterlocutor ? .leading : .trailing
+            self.messageText = messageText
         }
 
         var body: some View {
             VStack(alignment: horizontalAlignment, spacing: 0) {
-                Text("Hi! your last shot was realy good!")
+                Text(messageText)
```

Comment:
Hint: there is an extension that makes a `String` a view so in cases like those you can just write a string or use a string property. In other words `messageText` should be enough :)

Reply:
fixed